integration-tests/src/test/java/oracle/weblogic/kubernetes/annotations/AssumeWebLogicImage.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Hi Maggie, Since this is the only test using this annotation you can put these 2 methods directly in the test class. There is no need to add extra classes IMO.